tests/test-wrapper

```diff
@@ -48,4 +48,4 @@ for option in "${TEST_PARAMS[@]}"; do
 done
 
 echo "Executing ${COMMAND}"
-${COMMAND}
+${COMMAND} 2>&1 | tee "${TESTBLDDIR}/${TEST_NAME}-${TOKEN_DRIVER}.log"
```

Comment:
Hi Daiki,
do not worry about this, I already tested a correct solution (this one is broken, already been there) and will opena  PR for after we merge yours.

Reply:
OK, I removed this commit.